.github/workflows/regression-tests.yml

```diff
@@ -237,7 +237,7 @@ jobs:
       if: startsWith(matrix.os, 'windows')
       run: python -m pip install  --global-option build_ext --global-option --compiler=mingw32 -e .
     - name: Install Python testing dependencies (Windows)
-      run: pip install coverage xunitparser pytest pytest-cov
+      run: pip install coverage xunitparser pytest pytest-cov cocotb-bus
```

Comment:
I don't think this is a good idea. Any tools that depend on cocotb should be installed after cocotb itself.

Reply:
Oh, nevermind